Sources_v3/StreamChat/Models/Member.swift

```diff
@@ -12,7 +12,7 @@ import Foundation
 ///
 /// Learn more about using custom extra data in our [cheat sheet](https://github.com/GetStream/stream-chat-swift/wiki/StreamChat-SDK-Cheat-Sheet#working-with-extra-data).
 ///
-public typealias ChatChannelMember = _ChatChannelMember<NameAndImageExtraData>
+public typealias ChatChannelMember = _ChatChannelMember<DefaultExtraData.User>
```

Comment:
This was actually wrong all the time, right? :)

Reply:
yep, in many places we've used `NameAndImageExtraData` where we should've used `DefaultExtraData.User`, I've refactored them all (but only for Channel and User, obv)